src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -753,6 +768,7 @@ private void getContainerInfoBuilder(Offer offer, Request request, String slaveN
                   .newBuilder()
                   .setName("ports")
                   .setType(Value.Type.RANGES)
+                  .setRole(roleToUse)
```

Comment:
What would happen if there no role set at Mesos ? In case if it returns null will this affect this ?